src/Tools/dotnet-trace/dotnet-trace.csproj

```diff
@@ -2,28 +2,42 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-
-    <!-- Target .NET Core 2.1 so it will run on LTS -->
     <TargetFramework>netcoreapp2.1</TargetFramework>
+    <RootNamespace>Microsoft.Diagnostics.Tools.Trace</RootNamespace>
+    <IsPackable>False</IsPackable>
+    <PackAsTool>False</PackAsTool>
```

Comment:
You need a little more here to properly build and sign the dotnet CLI global tool (leave in the commented out part):

     <PackAsToolShimRuntimeIdentifiers>win-x64;win-x86;osx-x64</PackAsToolShimRuntimeIdentifiers>

    <!-- The package version needs to be hard coded as a stable version so "dotnet tool install -g dotnet-sos" works
    <Version>$(VersionPrefix)</Version>
    <PackageVersion>$(VersionPrefix)</PackageVersion>
     -->

    <ToolCommandName>dotnet-trace</ToolCommandName>
    <RootNamespace>Microsoft.Diagnostics.Tools.Trace</RootNamespace>
    <Description>Diagnostic trace tool</Description>
    <PackageTags>Diagnostic</PackageTags>
    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
    <!-- Need to put the shims here to sign -->
    <PackagedShimOutputRootDirectory>$(OutputPath)</PackagedShimOutputRootDirectory>